packages/builder/src/pages/builder/portal/settings/ai/index.svelte

```diff
@@ -56,10 +56,13 @@
     } else {
       // We don't store the default BB AI config in the DB
       delete fullAIConfig.config.budibase_ai
+
       // unset the default value from other configs if default is set
       if (editingAIConfig.isDefault) {
         for (let key in fullAIConfig.config) {
-          fullAIConfig.config[key].isDefault = false
+          if (key !== id) {
```

Comment:
I don't quite understand this change. The old code looks correct, what does this fix?

Reply:
Since it iterates through all the configs, It unsets the new one you actually want to set to default back to `false`, so it needs to know not to do that

Reply:
Ahhh damn, I missed that. Good catch.